build.gradle

```diff
@@ -1,15 +1,18 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext {
+        nav_version = "2.4.0"
```

Comment:
it's awesome to utilize this to have a consistency with versions across your application. in general, the only advice i'd give is just be consistent. i noticed some dependencies are hard coding the version, and some are utilizing these. it'd be nice to just align on usage

Reply:
Noted, thanks!

Reply:
I checked through the files and none of the other versions were repeated elsewhere in the code. I feel that it might be redundant to place versions for singular dependencies, but would that possibly change if it came to scaling a larger app and keeping it maintainable?

Reply:
yeah, it generally helps if you have modularization in mind. right now, you may only be using a lot of these dependencies once, but as your app grows, you might need different dependencies in other modules